### PR TITLE
Validate refund receiver in relayer

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: universal-login-relayer/lib/core/services/validators/RefundReceiverValidator.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import {IMessageValidator, SignedMessage, ensure} from '@universal-login/commons';
+import {Wallet} from 'ethers';
+import {InvalidRefundReceiver} from '../../utils/errors';
+
+export class RefundReceiverValidator implements IMessageValidator {
+  constructor(private wallet: Wallet) {
```
Comment (marekkirejczyk) [inline universal-login-relayer/lib/core/services/validators/RefundReceiverValidator.ts]: could be just address from config

File: universal-login-relayer/lib/core/services/validators/MessageExecutionValidator.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import {Wallet} from 'ethers';
+import {GasComputation} from '../../services/GasComputation';
+import {ComposeValidator} from '../../../core/services/validators/ComposeValidator';
+import {GasValidator} from './GasValidator';
+import {RefundReceiverValidator} from './RefundReceiverValidator';
+
+export class MessageHandlerValidator extends ComposeValidator {
```
Comment (xitronix) [inline universal-login-relayer/lib/core/services/validators/MessageExecutionValidator.ts]: Naming incosistant: file => MessageExecutionValidator, className => MessageHandlerValidator

File: universal-login-relayer/lib/core/services/validators/RefundReceiverValidator.ts (hunk 1 above)
Comment (JustynaBroniszewska) [reply universal-login-relayer/lib/core/services/validators/RefundReceiverValidator.ts]: True.. Done

File: universal-login-relayer/lib/core/services/validators/MessageExecutionValidator.ts (hunk 2 above)
Comment (JustynaBroniszewska) [reply universal-login-relayer/lib/core/services/validators/MessageExecutionValidator.ts]: ✅

**Pull request merged**